frontend/src/actions/index.tsx

```diff
@@ -17,4 +17,30 @@ export interface DecrementAction extends Action {
   type: CounterTypes.DECREASE_COUNT
 }
 
-export type CounterActions = IncrementAction | DecrementAction;
\ No newline at end of file
+export type CounterActions = IncrementAction | DecrementAction
+
+export const SET_USER = 'SET_USER'
+export const LOG_OUT = 'LOG_OUT'
+
+export interface UserDetailsOnRegister {
+  email: string
+  username: string
```

Comment:
- We didn't planned to use `username` on registration. So we can remove it and use a single interface.
 - Please, use prefix `I` to all interfaces